src/encoded/types/biosample.py

```diff
@@ -59,18 +73,54 @@ class Biosample(Item):  # CalculatedBiosampleSlims, CalculatedBiosampleSynonyms)
         'cell_culture_details.morphology_image.attachment.download',
         'cell_culture_details.morphology_image.attachment.width',
         'cell_culture_details.morphology_image.attachment.height',
+
+        # OntologyTerm linkTo
         'cell_culture_details.tissue.term_name',
         'cell_culture_details.tissue.preferred_name',
+        'cell_culture_details.tissue.slim_terms',
+        'cell_culture_details.tissue.synonyms',
+        'cell_culture_details.tissue.term_id',
+
+        # Modification linkTo
         'modifications.modification_type',
+        'modifications.genomic_change',
+        'modifications.target_of_mod',
```

Comment:
`target_of_mod` is a biofeature linkTo so I think you need either `modifications.target_of_mod.*` or more lines indicating which BioFeature properties.